clients/src/main/java/org/apache/kafka/common/Uuid.java

```diff
@@ -143,12 +143,6 @@ public int compareTo(Uuid other) {
             return 1;
```

Comment:
@divijvaidya  I could not find a prettier way to use `Long.compare` for `mostSignificantBits` , do you have a trick for it ?

Reply:
Something like this:
```
@Override
public int compareTo(Uuid other) {
    int res = Long.compare(mostSignificantBits, other.mostSignificantBits);
    return (res != 0) ? res : Long.compare(leastSignificantBits, other.leastSignificantBits);
}
```

Reply:
Thanks so much @divijvaidya,  looks great, I already updated the pull request

Reply:
Still there is another way to implement the `compareTo` method, and by the way `compare`  can be a pretty name also for the method, and it will keep the same style of the compare method name in `Long` 

 
       if (mostSignificantBits == other.mostSignificantBits)
            return Long.compare(leastSignificantBits, other.leastSignificantBits);
        return Long.compare(mostSignificantBits, other.mostSignificantBits);